lib/errors/silent.js

```diff
@@ -3,9 +3,12 @@
 var SilentError = require('silent-error');
 var deprecate   = require('../utilities/deprecate');
 
+// Get the call stack so we can let the user know what module is using the deprecated function.
+var stack = new Error().stack;
+
 Object.defineProperty(module, 'exports', {
   get: function () {
-    deprecate('`ember-cli/lib/errors/silent.js` is deprecated, use `silent-error` instead.', true);
+    deprecate("`ember-cli/lib/errors/silent.js` is deprecated, use `silent-error` instead.  Module: \n" + stack.toString(), true);
```

Comment:
So, the trace isn't from the "right place" but it actually contains the information that we need nonetheless since all of that is still in the same stack.

Is that "good enough" @stefanpenner?